src/main/java/org/squiddev/plethora/integration/forestry/MethodsErrorLogicSource.java

```diff
@@ -0,0 +1,22 @@
+package org.squiddev.plethora.integration.forestry;
+
+import forestry.api.core.IErrorLogicSource;
+import forestry.api.core.IErrorState;
+import forestry.core.config.Constants;
+import org.squiddev.plethora.api.method.BasicObjectMethod;
+import org.squiddev.plethora.api.method.IContext;
+import org.squiddev.plethora.utils.LuaList;
+
+public class MethodsErrorLogicSource {
+	@BasicObjectMethod.Inject(
+			value = IErrorLogicSource.class, modId = Constants.MOD_ID, worldThread = true,
+			doc = "function():table -- Get any errors preventing operation"
+	)
+	public static Object[] getErrors(IContext<IErrorLogicSource> context, Object[] arg) {
+		return new Object[] {
+				context.getTarget().getErrorLogic().getErrorStates().stream()
```

Comment:
I'm so going to convert this into a loop at some point :p. I'll live with it for now though.

Reply:
Yeah, this one may have been a bit gratuitous :stuck_out_tongue: